js-structure/js-structure_shapes/index.js

```diff
@@ -1,23 +1,9 @@
 console.clear();
+import { getRandomColor as randomColor } from "./utils/randomColor.js";
```

Comment:
Since you already imported the randomColor() function in all shape components, it is not needed here anymore.